api/schema.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/puppetlabs/wash/plugin"
+)
+
+var schemaHandler handler = func(w http.ResponseWriter, r *http.Request) *errorResponse {
```

Comment:
This should have annotations for API docs.

Reply:
I can add that once we agree on what the `EntrySchema` object should look like.

Reply:
Nevermind, I'll add all of that once `stree` + some `wash find` optimizations are finished.